### ISSUE TITLE
The Outer Worlds - Low Settings at 15 FPS

### ISSUE THREAD

**Issue by mightymos** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#905]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game [Required]

The Outer Worlds

### Game Platform [Required]

- [ ] Steam
- [ ] Microsoft Store
- [X] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [ ] Amazon Games app
- [ ] Other (please specify)

### Processor / Processor Number [Required]

AMD Ryzen 7 7700X

### Graphic Card [Required]

ASRock Intel Arc B580

### GPU Driver Version [Required]

32.0.101.6325_101.6252

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [X] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build [Required]

Windows 11 24H2

### Other Windows build

26100.2605

### Intel System Support Utility report

[intel_ssu_12-18-2024.txt](https://github.com/user-attachments/files/18190695/intel_ssu_12-18-2024.txt)

### Description and steps to reproduce [Required]

Start the game from GOG client.
This describes minimum DirectX 11 requirement, but not sure how to confirm renderer type actually used:
https://www.pcgamingwiki.com/wiki/The_Outer_Worlds

### Game graphic quality [Required]

- [X] Low
- [ ] Medium
- [ ] High
- [ ] Ultra

### Game resolution [Required]

1920x1080

### Game VSync [Required]

On

### Game display mode [Required]

- [X] Fullscreen
- [ ] Windowed
- [ ] Borderless Windowed

### Detailed game settings [Required]

![the_outer_worlds_32_0_101_6252_12-18-2024](https://github.com/user-attachments/assets/27c6bba1-5148-496d-bc99-3a2f900b1700)

### Save game

[Autosave00.zip](https://github.com/user-attachments/files/18190714/Autosave00.zip)

**Comment by Zack-Intel** [comment]
Hi @mightymos 

Thank you for the inquiry. I'll begin testing and get my results back to you soon.

In the meantime, can you clarify if the low fps is consistent, or triggered by anything specific in the game? Can you also clarify if this is the original or the Spacer's Choice edition?

**Comment by mightymos** [comment]
The GoG copy I have is for the original edition.
I just load into an old save game and immediately see poor performance.
I can obviously begin a new save file if that somehow benefits testing.
Choosing Detect Graphic Settings selects or leaves the settings at Low.

I do have the Spacers Choice edition from an Epic giveaway.
It seems to run well at 60 fps with a fresh save game.
Choosing Detect Graphic Settings selects High settings.

**Comment by IntelArcTesting** [comment]
I believe there is one specific setting in this game that causes that, believe it was screen space global illumination.

**Comment by Zack-Intel** [comment]
Thank you for the update, @mightymos 

We weren't able to replicate this issue as it looks like we can perform high settings for the original release of the game. We would like to compare the executable name to see if there's a performance difference between GOG and Steam versions of the game. Also, trying a different save file might not be a bad idea to rule out save file corruption.

@IntelArcTesting Is that feature exclusive to the Spacer's Choice Edition? I don't see that setting for the original and I'm only seeing stuff online where it it talked about for the re-release.

**Comment by mightymos** [comment]
Okay, I think the real game binary (not some kind of launcher which is named TheOuterWorlds.exe) is this one:
Indiana-Win64-Shipping.exe

Here are file checksums:

![Screenshot 2024-12-23 164703](https://github.com/user-attachments/assets/2d16d8a4-811a-4cb2-a66e-400eedfbfe66)

This is in folder Indiana/Binaries/Win64 along with file steam_appid.txt with contents 578650.
Again this is for sure the GoG installation.

I deleted all saves and started a fresh save with similar performance at low settings.

I also do not see a screen space global illumination setting.
I can continue providing information to attempt to isolate the issue on my machine.

I believe a new Intel Arc driver version was released.

**Comment by IntelArcTesting** [comment]
@Zack-Intel it was the spacers choice from epic games, I found my old Reddit post again and at that time I had black screen issue and crashes right after loading in that was caused by global illumination, not low fps. Thought it was my bad.

**Comment by Zack-Intel** [comment]
Hi @mightymos 

While I'm still not able to reproduce, I suspect it may potentially have something to do with the graphics going to the iGPU instead given our discrepancies. I would double check some BIOS settings and make sure the iGPU is disabled (alongside enabling rebar and disabling CSM), and do a clean driver reinstall.

**Comment by mightymos** [comment]
@Zack-Intel Good catch, I disabled the iGPU and hybrid graphics for good measure. Rebar was enabled, I honestly didn't check on CSM.

The game runs normally on Very High settings now.
I imagine the issue can be closed now unless anything else needs to be checked.

**Comment by Zack-Intel** [comment]
Glad to hear @mightymos 

If you run into any further issues, feel free to open a new ticket and we will continue assisting.
